comments/models.py

```diff
@@ -0,0 +1,37 @@
+from django.db import models
+from django.forms import CharField
```

Comment:
불필요한 package가 Import 된 것 같습니다. 제거해 주세요.
```suggestion
from django.db import models
```